jobs/integr8ly/ocp3/pds/pds-testing-executor.yaml

```diff
@@ -177,9 +180,15 @@ This needs to be changed to 'false' for Integreatly Workshop since it uses valid
 
         // Waiting for ${sleepTime} minutes so that resources scheduled for termination are actually terminated
         def uninstall(sleepTime) {
+
+            String uninstallationBranch = params.INSTALLATION_BRANCH
+            if (params.UNINSTALLATION_BRANCH) {
```

Comment:
Isn't this statement always true (even if the param is left empty)?

Reply:
I tried here, it seems it is false if UNINSTALLATION_BRANCH is left empty:

https://integreatly-qe-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/view/all/job/trepel-pipeline-playground/46/

see it's params, config and console output. It outputted BUILD_NUMBER since YOURCITY was left empty.